tests/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from unittest import mock
+from fakeredis import FakeStrictRedis
+
+
+@pytest.fixture
+def redis_mock(monkeypatch):
```

Comment:
why would we want to mock redis?

Reply:
Because it's easy :) but more seriously, it's a step on the way to being able to run tests without manually starting a server first, it gives us at least something that can be run in CI (`test_ready_endpoint`)